mkosi/__init__.py

```diff
@@ -7753,6 +7753,13 @@ def run_qemu_cmdline(config: MkosiConfig) -> Iterator[List[str]]:
             "-append", build_auxiliary_output_path(config, ".cmdline").read_text().strip(),
         ]
 
+    if swtpm_socket is not None:
+        cmdline += [
+            "-chardev", f"socket,id=chrtpm,path={swtpm_socket}",
+            "-tpmdev", "emulator,id=tpm0,chardev=chrtpm",
+            "-device", "tpm-tis,tpmdev=tpm0",
+            ]
+
     with contextlib.ExitStack() as stack:
```

Comment:
You can call it in this block as follows:

```
socket = stack.enter_context(start_swtpm())
```

and then add it to the cmdline right after that

Reply:
i don't grok what you are saying? i guess my python-fu is too limited...